packages/vuetify/src/components/VList/VListItem.ts

```diff
@@ -104,16 +111,32 @@ export default baseMixins.extend<options>().extend({
 
       this.to || this.toggle()
     },
+    genAttrs () {
+      const attrs: Record<string, any> = {
+        'aria-disabled': this.disabled ? true : undefined,
+        'aria-selected': String(this.isActive),
+        tabindex: this.isClickable && !this.disabled ? 0 : -1,
+        ...this.$attrs,
+      }
+
+      if (this.$attrs.role) {
+        // do nothing, role already provided
```

Comment:
maybe return `attrs` in here for a little optimization?

Reply:
If it hits this conditional, the rest are not evaluated. I think it helps the readability that attrs isn't returned in more than one place (at least in this situation).

Reply:
You're right, returning things as soon as possible it's a pattern that I'm used to follow, but in this case does not make much sense.